internal/provider/provider.go

```diff
@@ -90,6 +97,8 @@ func (p *OCIProvider) Configure(ctx context.Context, req provider.ConfigureReque
 		opts.defaultExecTimeoutSeconds = *data.DefaultExecTimeoutSeconds
 	}
 
+	opts.skipExecTests = p.skipExecTests || (data.SkipExecTests != nil && *data.SkipExecTests)
```

Comment:
can we also plumb this through as an environment variable for runtime skips?

I know you can use `TF_VAR_...` to plumb things through that way, but for good measure a OCI_EXEC_TEST_SKIP` or something that isn't subject to the same var plumbing would be helpful imo

Reply:
Do we have a precedent for that in imagetest that I could draft on?

Also: how do you feel about this as a followup PR?

Reply:
yeah I'm thinking just something trivial like so: https://github.com/chainguard-dev/terraform-provider-imagetest/blob/eec749b498bd39034a1b8125e790ee58bb968bf1/internal/provider/provider.go#L323-L325

but yeah, I approved, its fine in a follow up!

Reply:
https://github.com/chainguard-dev/terraform-provider-oci/pull/196